deck-o-cards.js

```diff
@@ -1,49 +1,21 @@
-// 1. Make the function deck_o_cards assemble an array of cards using the provided suits and values arrays.
-// Each card in the deck should be an object formatted as: {suit: 'hearts', value: 'A'}
 function deck_o_cards() {
-  var values = [2, 3, 4, 5, 6, 7, 8, 9, 10, 'Jack', 'Queen', 'King', 'Ace'];
-  var suits = ['hearts', 'diamonds', 'clubs', 'spades'];
-
-  var cards = []; // deck
-  var shuffledCards = []; // deck shuffled
-
-  // Make 52 card objects and store them in the "cards" array
-  // Hint: use 2 for loops
-
-
-  // 2. Shuffle the cards
-  // Hint: shuffle function is already defined below
-
-
-  // Pull the top card from the newly shuffledCards
-
-
-  // 3. Print the results:
-  // "The deck has {n} cards"
-  // "The top card is the {value} of {suit}"
-
+	var deck = [];
+	var values = [2, 3, 4, 5, 6, 7, 8, 9, 10, 'Jack', 'Queen', 'King', 'Ace'];
+	var suits = ['hearts', 'diamonds', 'clubs', 'spades'];
+	for (i = 0; i < suits.length; i++) {
+		 var suit = suits[i];
+		 for (j = 0; j < values.length; j++) {
+		 var value = values[j];
+		 var card = {
+			suit: suit,
+			value: value
+			}
```

Comment:
Alignment is off here. This is the closing `}` to the `card` object, but it's hard to tell since it's indented in